src/Views/Dashboard/Workspaces/Presentations.js

```diff
@@ -2,13 +2,72 @@ import React from "react";
 import { css, withStyles } from "../../../withStyles";
 import FlexContainer from "../../../Containers/FlexContainer";
 import Heading from "../../../Elements/Heading";
+import Paragraph from "../../../Elements/Paragraph";
+import List from "../../../Elements/List";
+import ListItem from "../../../Elements/ListItem";
+import ButtonLink from "../../../Elements/ButtonLink";
 
 const Presentations = ({ styles, ...props }) => {
-  return <Heading size="3">Presentations</Heading>;
+  console.log(props.data);
+  return (
+    <FlexContainer fullWidth="1">
+      <div {...css(styles.presentationsInfo)}>
+        <Paragraph size="sub">
+          Total presentations held in this workspace:{" "}
+          {props.data.presentations.length}
+        </Paragraph>
+      </div>
+      <div {...css(styles.presentations)}>
+        <Paragraph style={{ margin: "0" }} size="sub">
+          Your latest presentations in this workspace:
+        </Paragraph>
+        <List style={{ width: "100%" }}>
+          {props.data.presentations.map((p, i) => {
+            if (i > 4) return;
+            return (
+              <div {...css(styles.listItems)}>
+                <ListItem
+                  key={i}
+                  iconPosition="right"
+                  iconFillColor="darkMetal"
+                >
+                  <Paragraph size="sub">{p.date}</Paragraph>
+                  <Paragraph size="sub">Name: {p.name}</Paragraph>
+                  <Paragraph size="sub">Attendees: {p.attendees}</Paragraph>
+                </ListItem>
+              </div>
+            );
+          })}
```

Comment:
This will still loop over all the items despite the return-statement (even though it's not going to return the remaining items).

I believe we can improve it by doing something like this instead:

```jsx
{
    props.data.presentations.some((p, i) => {
       if (i === 4) return true;

       // code...
    });
}
```